specs/agents/tracing-instrumentation-aws.md

```diff
@@ -53,10 +53,16 @@ For distributed tracing, the SQS API has "message attributes" that can be used i
 ### SNS (AWS Simple Notification Service)
 
 The AWS Simple Notification Service can be instrumented using the [messaging spec](tracing-instrumentation-messaging.md), 
-but the only action that is instrumented is `PUBLISH`. These specifications supersede those of the messaging spec: 
+but the only action that is instrumented is `PUBLISH`. These specifications supersede those of the messaging spec:
 
 - `span.name`: The span name should follow this pattern: `SNS PUBLISH <TOPIC-NAME>`. For example,
```

Comment:
As long as we maintain consistency with the messaging spec, should the name be: `SNS PUBLISH to <TOPIC-NAME>`?

Reply:
https://github.com/elastic/apm/blob/master/specs/agents/tracing-instrumentation-messaging.md#naming says the name should be `<MSG-FRAMEWORK> SEND/RECEIVE/POLL to/from <QUEUE-NAME>`, so should the name here be `SNS SEND to <TOPIC-NAME>`?

Reply:
From team discussion earlier today it was decided to use `SNS PUBLISH to <TOPIC-NAME>` (with `span.action = 'publish'`). According to https://github.com/elastic/apm/issues/446, Ruby is already doing this. Go doesn't have the " to".